C++.en.md

```diff
@@ -0,0 +1,146 @@
+Wiren Board C++ Style Guide
+========================
+
+The following sheet is based on Yandex code style for C++. If any guideline is missing from the following document, 
+please follow Yandex official documentation: https://clickhouse.yandex/docs/en/development/style.html . But if you
+see any collision between documents, please follow the guide below. 
+
+## Marking
+
+For identation blocks only spaces are used and 4 spaces count as 1 tab.
+
+Aritmetic operations are always separated by spaces:
+```C++
+int sum = a + b;
+```
+Whitspace should be placed before each bracket opening.
+```C++
+int function (int a, int b, int c) 
+{
+  int d = abs ( (a * sqrt (b)) + c);
+  return d;
+}
+```
+
+At sequence of variables comma should be placed right after variable name and the comma should be followed by space before
+next variable name:
+
+```C++
+int sum = GetSum (a, b);
+```
+
```

Comment:
the same goes for every punctuation mark, whether it's comma or semicolon or colon or whatever else

Reply:
Are you sure it should be used for all? It would result something like this:
if (! variable) { }
instead of:
If ( !variable) { }